app/views/layouts/_authoring.html.erb

```diff
@@ -6,7 +6,11 @@
           <%= render partial: 'layouts/copyright' %> -
         </span>
       <% end %>
-      <%= raw t :authoring_note, authors: guide.authors, collaborators: "https://raw.githubusercontent.com/#{guide.slug}/master/COLLABORATORS.txt" %>
+      <% if guide.collaborators.present? %>
+        <%= raw t :authoring_note_with_collaborators, authors: guide.authors, collaborators: "https://raw.githubusercontent.com/#{guide.slug}/master/COLLABORATORS.txt" %>
+      <% else %>
+        <%= raw t :authoring_note, authors: guide.authors %>
```

Comment:
The plain `authoring_not` does not require the usage of `raw` helper, right?

Reply:
Without `raw` the authoring note looks like

![image](https://user-images.githubusercontent.com/11304439/56518059-d6dbef80-6514-11e9-81b6-5a44ee747266.png)

instead of

![image](https://user-images.githubusercontent.com/11304439/56518034-c88dd380-6514-11e9-8fb5-17f3aac9c247.png)

Reply:
may I suggest https://makandracards.com/makandra/52455-rails-including-html-in-your-i18n-locales?

Reply:
yes you may. I'll add the html suffix